Jenkinsfile

```diff
@@ -52,9 +52,14 @@ def presubmit(gitUtils, bazel, utils) {
     }
     stage('Integration Tests') {
       timeout(15) {
-        sh('bin/e2e.sh -tag alpha' + gitUtils.GIT_SHA + ' -v 2')
+        sh("bin/e2e.sh -tag alpha${gitUtils.GIT_SHA} -v 2")
       }
     }
+    stage('Docker Test Push'){
+      def images = 'init,runtime'
+      def tags = gitUtils.GIT_SHA
+      utils.publishDockerImagesToContainerRegistry(images, tags)
```

Comment:
(the assumption in e2e is that cluster is not local, so we move images to a central repo first)